### ISSUE TITLE
GPU training time?

### ISSUE THREAD

**Issue by PetrochukM** [Rayhane-mamah/Tacotron-2#21]
Hi There!

How long does it take to train Tacotron-2?

Thanks!

**Comment by Rayhane-mamah** [comment]
Hello again,

As for training, a training step takes 0.75 seconds (with a batch size of 64). It takes around 12 hours to do 60k steps.

It takes about few thousand steps to get a perfect feature prediction model so I expect the training to last for 24~36 hours. I'll let it training longer just to check if model keeps improving.

Actual stats at 60k steps:
![screenshot from 2018-04-14 10-49-43](https://user-images.githubusercontent.com/34689728/38766951-9b076700-3fd1-11e8-9aac-22251ab7f830.png)

As long as loss will be decreasing significantly I'll let it train.

Note:
Again, these time stats are provided using a Titan XP GPU.

**Comment by eyaler** [comment]
@Rayhane-mamah training ljspeech on v100 (tf 1.12, cuda10, google vm image) with batch=32, i am getting ~5 sec/step. how can i go about debugging this?

**Comment by shaktikshri** [comment]
I am also seeing the same time of 5 to 6sec/step. I am on a V100 GPU with batch size 32. Is there any suggestions in general?
